### DISCUSSION TITLE
Subscribing to a podcast suddenly no longer works

### DISCUSSION THREAD

**Discussion by DJ-SteffEN** [ampache/ampache#3974] [Q&A]
Hello,

I have added several new podcasts, the result is always the same. 
The folders in the path and the episodes in the DB are created with the status 'Pending', nothing more.
No downloads appear in the folders, a subsequent synchronization does not help either.

@([inthemix.at](https://inthemix.at/))
Ampache version: 5.4.0-release
![image](https://github.com/ampache/ampache/assets/111641714/881cffda-3327-44d6-b12b-555846141c33)

There are no entries in the log file with my default settings.

What can I do, or where is the problem?

Thank you for your help
DJ Steffen

**Comment by lachlan-00** [comment]
it gets the episodes but doesn't download?

can you manually sync an episode with the sync button?

**Comment by DJ-SteffEN** [comment]
Hello lachlan-00,

yes, the sync button for the podcast does not work, except for a short ajax-loading in the header.
However, when I press the sync button for the podcast episode, the episode is downloaded.
I have now synchronized all 10 episodes individually.
Unfortunately, as before, the status of all episodes remains Pending and they cannot be played.

Thank you for your help
DJ Steffen

![image](https://github.com/ampache/ampache/assets/111641714/46769098-92c2-4a1f-b098-fd60ede0ed38)

![image](https://github.com/ampache/ampache/assets/111641714/aad7ee9b-167e-4355-a2ef-aeb911df649e)

**Reply by lachlan-00** [reply]
that's an odd bug. the episode download should change the status.

you said 5.4? i do have a 5.6 server for testing at https://5.ampache.dev

might be able to ID somethign from that. can you share the podcast link and i'll add it there.

**Comment by DJ-SteffEN** [comment]
Super thanks,
the podcast link is: `https://cafemamboibiza.com/podcast.xml`

By the way
Externally I can play MP3 normally, in Ampache not even if I have previously set the episode to 'completed' in the DB.

**Comment by DJ-SteffEN** [comment]
Hello lachlan-00,

I have fixed the problem, the fault was mine.
I had tried something some time ago and did not turn it back completely.

Thanks again
